lib/foreman_maintain/runner.rb

```diff
@@ -92,7 +92,6 @@ def storage
 
     def execute_scenario_steps(scenario, force = false)
       scenario.before_scenarios.flatten.each { |before_scenario| run_scenario(before_scenario) }
-      confirm_scenario(scenario)
```

Comment:
As per you mentioned, it is added for @rescue_scenario. Instead of removing, would it be possible to add condition here which checks if scenario is equal to @rescue_scenario?

Reply:
Added the change.
Thank you !

Reply:
@upadhyeammit, thanks for adding condition which checks for presence of @rescue_scenario.

Currently, `rescue_scenario` is used by only backup_command.
Not sure but I think, the condition that you added is not enough as `run_scenario` method gets invoked by [backup_command](https://github.com/theforeman/foreman_maintain/blob/master/lib/foreman_maintain/cli/backup_command.rb#L25) where it passes backup_scenario & rescue_scenario as arguments.

There is no any before_scenarios i.e preparation steps for `backup_scenario` argument.
While running this backup_scenario, it will invoke [execute_scenario_steps](https://github.com/theforeman/foreman_maintain/blob/master/lib/foreman_maintain/runner.rb#L51) from `run_scenario`. In this case, the condition that you have added will pass successfully.

So I just want to make sure for `backup_scenario` execution, it should not ask confirmation twice and this confirmation should invoke only when `scenario` inside `execute_scenario_steps` matches with the @rescue_scenario.

Reply:
I tested this and I dont see confirm call twice for backup and rescue scenarios.